narwhals/expr.py

```diff
@@ -1715,7 +1837,8 @@ def sort(self, *, descending: bool = False, nulls_last: bool = False) -> Self:
         return self.__class__(
             lambda plx: self._to_compliant_expr(plx).sort(
                 descending=descending, nulls_last=nulls_last
-            )
+            ),
+            is_order_dependent=True,
```

Comment:
We removed `sort` anyway. But.. is `sort` order dependent?

Reply:
probably 😄 but yes, if it's being removed it may not matter